index4_7.js

```diff
@@ -0,0 +1,22 @@
+/*7.	Написать свою реализацию метода массива map (myMap) которая работает точно так же как существующая. ВСЕ массивы должны иметь этот метод. Примеры ниже.
+
+const numbers = [1,2,3]
+	
+	const result1 = numbers.myMap((number) => number * 2) // [2,4,6]
+	const result2 = numbers.myMap((number) => String(number)) // [‘1’, ‘2’, ‘3’]
+*/
+
+Array.prototype.myMap = function (cb) {
+  let result = [];
+  for (let item of this) {
```

Comment:
your map function is not equal to original one. No access to index and currentArray provided